packages/core/src/config/database.ts

```diff
@@ -28,10 +28,15 @@ export const buildDatabase = ({
   config: Config;
 }): DatabaseConfig => {
   const { ponderDir, rootDir } = common.options;
-  const defaultStorePath = path.join(ponderDir, "store");
-  const defaultSyncFilePath = path.join(defaultStorePath, "sync.db");
-  const defaultIndexingFilePath = path.join(defaultStorePath, "indexing.db");
-  const sqlitePrintPath = path.relative(rootDir, defaultStorePath);
+  const storePath = (() => {
+    if (config.database?.kind === "sqlite" && config.database.filename) {
+      return path.join(rootDir, config.database.filename);
+    }
+    return path.join(ponderDir, "store");
+  })();
```

Comment:
IMO a ternary here is probably more idiomatic for TS:
```ts
const storePath = config.database?.kind === "sqlite" && config.database.filename
  ? path.join(rootDir, config.database.filename);
  : path.join(ponderDir, "store");
```

Reply:
good point